docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
Actually, we did consider coordinator , daemon, and epic...I really liked epic since it's similar in meaning to saga but still fairly novel in CS (besides the agile term)..but we always had (relatively weak) excuses for not choosing them. I'm at the point where I just want almost _anything_ as long as the community mostly doesn't hate it. At least in this thread, it sounds like daemon is liked by many. Any opinions @blesh ?